### PR TITLE
feat(api): xroad configuration for offical journal service

### PR DESCRIPTION
**Pull request by jonbjarnio**
# ...

## What

Adding xroad configuration for the offical journal of iceland service

## Why

For better and safer communications

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

## Summary by CodeRabbit

- **New Features**
	- Integrated the Official Journal of Iceland into the list of services, enhancing access to official publications directly through the app.
- **Enhancements**
	- Improved the configuration setup for development, staging, and production environments to support the new Official Journal of Iceland service.
	- Increased the timeout for fetching data from the Official Journal of Iceland to improve reliability under slow network conditions.
- **Refactor**
	- Updated internal configurations and parameters for better alignment with the new service integration.

### CODE REVIEW

File: libs/clients/dmr/src/lib/dmrClient.config.ts [hunk 1]
```diff
@@ -13,9 +13,12 @@ export const DmrClientConfig = defineConfig<z.infer<typeof schema>>({
...
   schema,
   load: (env) => ({
-    basePath: 'https://api.official-journal.dev.dmr-dev.cloud',
+    xRoadServicePath: env.required(
+      'XROAD_OFFICIAL_JOURNAL_PATH',
+      'IS-DEV/GOV/10014/DMR-Protected/official-journal',
+    ),
     fetch: {
-      timeout: 10000,
+      timeout: 30000,
```
Comment (thorkellmani) [inline libs/clients/dmr/src/lib/dmrClient.config.ts:21]: Slow service?

Comment (jonbjarnio) [reply libs/clients/dmr/src/lib/dmrClient.config.ts:21]: Nei líklegast ekki lækka þetta.

**Review by thorkellmani — APPROVED** [review]
LGTM, just add the auth as soon as possible

**Review by robertaandersen — APPROVED** [review]
LGTM

**Pull request merged**